app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -222,8 +224,7 @@ export default function createRPCMethodTrackingMiddleware({
   ) {
     const { origin, method, params } = req;
 
-    const rateLimitType =
-      RATE_LIMIT_MAP[method] ?? RATE_LIMIT_TYPES.RANDOM_SAMPLE;
```

Comment:
why did we remove the `RATE_LIMIT_TYPES.RANDOM_SAMPLE` fallback?

Reply:
the switch has a default condition that falls back to the RANDOM_SAMPLE anyhow, so this is redundant